diff -Naur pjd-fstest-20080816.orig/tests/truncate/00.t pjd-fstest-20080816.new/tests/truncate/00.t
--- pjd-fstest-20080816.orig/tests/truncate/00.t	2008-03-29 06:43:55.000000000 -0700
+++ pjd-fstest-20080816.new/tests/truncate/00.t	2009-12-23 14:42:24.000000000 -0800
@@ -22,7 +22,7 @@
 expect 567 lstat ${n0} size
 expect 0 unlink ${n0}
 
-dd if=/dev/random of=${n0} bs=12345 count=1 >/dev/null 2>&1
+dd if=/dev/urandom of=${n0} bs=12345 count=1 >/dev/null 2>&1
 expect 0 truncate ${n0} 23456
 expect 23456 lstat ${n0} size
 expect 0 truncate ${n0} 1
